src/App.css

```diff
@@ -1,42 +1,48 @@
-#root {
-  max-width: 1280px;
-  margin: 0 auto;
-  padding: 2rem;
-  text-align: center;
+body {
+  background-color: #264653 !important;
```

Comment:
important niestety jest kategorycznie zabroniony :P dlaczego akurat na body go wymagasz, coś nie działa?

Reply:
To kod z googla przeklejony, nie moje style docelowe, skupiłam się na logice, na style przyjdzie czas, tu po prostu copy paste z konfiguracji auth0 zeby testować oauth. Nie uzywam importanta nigdy :p

Reply:
Nawet jeżeli to chwilowe rozwiązanie, to usunięcie importantów wiele nie kosztuje bo źle wygląda i zwykle blokuje code-review.